roles/baseline/tasks/main.yaml

```diff
@@ -45,6 +45,21 @@
   tags:
     - baseline
 
+- name: Lookup K8s version info
+  block:
+    - name: Retreive K8s cluster information
+      community.kubernetes.k8s_cluster_info:
+        kubeconfig: "{{ KUBECONFIG }}"
+      register: cluster_info
+    - name: Set the K8s server version
+      set_fact:
+        K8S_VERSION: "{{ cluster_info.version.server.kubernetes.major + '.' + cluster_info.version.server.kubernetes.minor }}"
+    - name: Output K8S_VERSION
+      debug:
+        msg: "{{ K8S_VERSION }}"
```

Comment:
Not really needed here? Also is this generic across all cloud providers?

Reply:
I can remove it if not wanted, I had seen that pattern used for some other set_facts in DAC, just thought it might be useful data in DAC output. This change will be tried with each of the cloud providers prior to asking for final approval based on your earlier suggestion.

Reply:
The fact is only used by ansible so it's not needed for anyone to see. Please remove.

Reply:
removed it.